nss-day-cohort-29/reactive-nutshell-copperden
# Meg chat edit

# Description
Added "Edit" functionality to each message. Also set scrollbar to display at bottom on page load.

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout meg-chat-edit`
3. Add the below JSON to your database.
3. Click on "Messages"
4. Check that the scrollbar is down at the bottom (the newest message should be visible). 
5. Click on any message text and an input field should replace the text, with the previous value in it.
6. Update the message text and hit enter - the field should be replaced with static text, showing your change.

```
"messages": [
    {
      "id": 1,
      "message": "User1 Message Content",
      "timeDisplay": "1/22/2019 at 3:18 PM",
      "timestamp": 1548710316504,
      "userId": 1
    },
    {
      "id": 2,
      "message": "User2 Message Content",
      "timeDisplay": "1/27/2019 at 3:18 PM",
      "timestamp": 1548710316504,
      "userId": 2
    },
    {
      "message": "test 1233456789",
      "timeDisplay": "1/28/2019 at 8:28 PM",
      "timestamp": 1548728880201,
      "userId": 1,
      "id": 5
    },
    {
      "message": "blah",
      "timeDisplay": "1/28/2019 at 8:28 PM",
      "timestamp": 1548728908179,
      "userId": 1,
      "id": 6
    },
    {
      "message": "yoooooo",
      "timeDisplay": "1/28/2019 at 8:29 PM",
      "timestamp": 1548728996414,
      "userId": 1,
      "id": 8
    },
    {
      "message": "what the heckin heck????",
      "timeDisplay": "1/28/2019 at 8:37 PM",
      "timestamp": 1548729442466,
      "userId": 1,
      "id": 9
    },
    {
      "message": "doot doot",
      "timeDisplay": "1/28/2019 at 9:12 PM",
      "timestamp": 1548731579780,
      "userId": 1,
      "id": 10
    },
    {
      "message": "lolol",
      "timeDisplay": "1/28/2019 at 9:39 PM",
      "timestamp": 1548733183905,
      "userId": 1,
      "id": 11
    },
    {
      "message": "this is just a test!!!",
      "timeDisplay": "1/29/2019 at 1:00 PM",
      "timestamp": 1548788445857,
      "userId": 1,
      "id": 12
    },
    {
      "message": "what the heck?!?!?!",
      "timeDisplay": "1/29/2019 at 1:07 PM",
      "timestamp": 1548788855331,
      "userId": 1,
      "id": 14
    }
  ]
```

# Checklist:
